.ci/teardown.sh

```diff
@@ -22,30 +22,43 @@ log_copy_dest="$1"
 runtime_log_location="/var/lib/clear-containers/runtime/runtime.log"
 runtime_log_filename="cc-runtime.log"
 runtime_log_path="${log_copy_dest}/${runtime_log_filename}"
+runtime_log_prefix="cc-runtime_"
 
 proxy_log_filename="cc-proxy.log"
 proxy_log_path="${log_copy_dest}/${proxy_log_filename}"
+proxy_log_prefix="cc-proxy_"
 
 shim_log_filename="cc-shim.log"
 shim_log_path="${log_copy_dest}/${shim_log_filename}"
+shim_log_prefix="cc-shim_"
 
 crio_log_filename="crio.log"
 crio_log_path="${log_copy_dest}/${crio_log_filename}"
+crio_log_prefix="crio_"
 
 # Copy log files if a destination path is provided, otherwise simply
 # display them.
 if [ ${log_copy_dest} ]; then
-	sudo cp "${runtime_log_location}" "${runtime_log_path}"
-	sudo journalctl --no-pager -u cc-proxy > "${proxy_log_path}"
-	sudo journalctl --no-pager -t cc-shim > "${shim_log_path}"
-	sudo journalctl --no-pager -u crio > "${crio_log_path}"
+	# Create the log files
+	cp "${runtime_log_location}" "${runtime_log_path}"
+	journalctl --no-pager -u cc-proxy > "${proxy_log_path}"
+	journalctl --no-pager -t cc-shim > "${shim_log_path}"
+	journalctl --no-pager -u crio > "${crio_log_path}"
+
+	# Split them in 5 MiB subfiles to avoid too large files.
+	pushd ${log_copy_dest}
+	split -b 5242880 -d ${runtime_log_path} ${runtime_log_prefix}
```

Comment:
Can we add a var for the 5242880 at the top of the file pls - easier to edit later, saves the duplication here.